src/services/fetchLocations.js

```diff
@@ -1,16 +1,25 @@
-/* eslint-disable import/prefer-default-export */
-import axios from "axios";
+const prepareParams = obj =>
+  Object.keys(obj)
+    .map(k => `${k}=${encodeURIComponent(obj[k])}`)
+    .join("&");
+
+const params = {
+  access_token: process.env.VUE_APP_MAPBOX_TOKEN,
+  language: "no",
+  country: "no",
+  autocomplete: true
+};
 
 export default async function fetchLocation(location) {
-  return axios
-    .get(`https://api.mapbox.com/geocoding/v5/mapbox.places/${location}.json`, {
-      params: {
-        access_token: process.env.VUE_APP_MAPBOX_TOKEN,
-        language: "no",
-        country: "no",
-        autocomplete: true
-      }
-    })
-    .then(response => response.data.features)
-    .catch(error => console.log(error));
+  try {
+    const resp = await fetch(
+      `https://api.mapbox.com/geocoding/v5/mapbox.places/${location}.json?${prepareParams(
+        params
+      )}`
+    );
+    return (await resp.json()).features;
+  } catch (err) {
+    console.error(err);
+    return null;
```

Comment:
Kanskje la Sentry plukke opp feilen fremfor å la den feile i det stille med en feilmelding hos klienten?

Reply:
Ah sant, vi har sentry. Kan jeg bare throwe den videre og la den plukke opp ved rota?

Reply:
Tror ikke det har blirr lagt til noe key ennå, så det kjører ikke :/ Må bare legges til i env for prod

Sentry skal plukke opp alle exceptions som blir kastet i komponenter